src/game_api/script/lua_vm.cpp

```diff
@@ -2096,6 +2100,53 @@ end
     /// Get engine target frametime when game is unfocused (1/framerate, default 1/33).
     lua["get_frametime_unfocused"] = get_frametime_inactive;
 
+    /// Destroys all layers and all entities in the level. Usually a bad idea, unless you also call create_level and spawn the player back in.
+    lua["destroy_level"] = destroy_level;
+
+    /// Destroys a layer and all entities in it.
+    lua["destroy_layer"] = destroy_layer;
+
+    /// Initializes an empty front and back layer that don't currently exist. Does nothing(?) if layers already exist.
+    lua["create_level"] = create_level;
+
+    /// Initializes an empty layer that doesn't currently exist.
+    lua["create_layer"] = create_layer;
+
+    /// Setting to false disables the death screen from popping up for any usual reason, can still load manually
+    lua["set_death_enabled"] = set_death_enabled;
```

Comment:
Ok so this also disables some small things like shop zoom, but do we care? I made it just for overlunking, but it's nice to have debugging tools in the api too.

Reply:
It's also just nice to not need a player in a level at all

Reply:
Yes for cutscenes or weird game modes in mods too I suppose. I'll change the name to something more descriptive, any ideas?

Reply:
Dunno, also going back to the patterns, we could just get the address from vtable of state.screen_level, and for the theme stuff from level_gen 
since they never gets destroyed and stuff

Reply:
I don't really want to redo the hooks for a fourth time, but you can suggest your fix in pr format and this works for now.